### PR TITLE
Add manual-test-voyager and manual-test-sweep AI QA skills

### PR DESCRIPTION
**Pull request by makalaaneesh**
### Describe the changes in this pull request

Adds two AI "skills" under `ai/skills/` that let a coding agent act as a manual QA engineer for `yb-voyager`. No product code is touched — this is agent tooling (markdown + helper shell scripts), in the same family as the existing `branch-review` / `pr-review-sweep` skills.

- **`manual-test-voyager`** — a 4-phase loop: (1) build a PR-scoped **test plan** from the diff (template covering manual + automated flows, biased to negative/edge cases and end-to-end state transitions), (2) **provision** live PG→YB (Docker by default; local/external fallback), (3) **execute** offline + live migrations under `timeout` and apply anomaly oracles (hang, crash/panic, silent data loss via live source↔target diff, sequence-advance, report parity, usability), (4) **report** and grow a regression library. Ships reference docs (runbook, environment-setup, validation-oracles, flag-surface, test-plan-template), a seeded regression library (adaptive-parallelism, partitions, sequences, status-commands, multi-schema), and provision/build/teardown scripts.
- **`manual-test-sweep`** — sweeps recent open PRs lacking an `[AI-manual-test]` comment, delegates each to `manual-test-voyager` in a sequential subagent, and posts one PR conversation comment. Mirrors `pr-review-sweep` → `branch-review`. Default depth is plan-only; `--execute` / `--execute --live` gate the heavier provision/build/migrate path. Uses the `[AI-manual-test]` marker (deliberately not `[AI]`) so it stays independent of `pr-review-sweep`'s idempotency check.

### Describe if there are any user-facing changes

None. These skills are agent-facing assets under `ai/skills/` and are not part of the shipped `yb-voyager` binary. No changes to the CLI, configuration, installation, or reports.

### How was this pull request tested?

Both flows the skill drives were validated end-to-end against Docker PostgreSQL 17 + YugabyteDB 2025.2.0.0:
- **Offline** — full pipeline (export/import schema+data), including the adaptive-parallelism boundary case `--adaptive-parallelism balanced --adaptive-parallelism-max 1`: no hang, parallelism clamped, full source↔target parity, sequences advanced. Negative controls (truncate-without-start-clean, parallel-jobs+adaptive, unreachable host) reject fast with actionable messages.
- **Live** — snapshot + CDC delta + `initiate cutover to target` (COMPLETED) + source↔target parity + `end migration` (clean replication-slot teardown).

No unit/integration tests are added: the change is docs + helper scripts with no product runtime surface. The provision/build/teardown scripts were executed directly during validation.

### Does your PR have changes in callhome/yugabyted payloads? If so, is the payload version incremented?

No.

### Does your PR have changes to on-disk structures that can cause upgrade issues?

No.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: ai/skills/manual-test-voyager/scripts/build-voyager.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# — this script builds exactly that. We print the real worktree HEAD below for the record.
+#
+# For LIVE / Debezium testing use the installer instead (see references/environment-setup.md):
+#   yes | bash installer_scripts/install-yb-voyager --install-from-local-source --only-pg-support
+#
+#   OUT=<path>   output binary path (default: <repo>/manual-test-runs/bin/yb-voyager)
+set -euo pipefail
+
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+OUT="${OUT:-${REPO_ROOT}/manual-test-runs/bin/yb-voyager}"
```
Comment (makalaaneesh) [inline ai/skills/manual-test-voyager/scripts/build-voyager.sh:20]: [AI] **Default build output lands inside the repo working tree and is not gitignored.**

`OUT` defaults to `${REPO_ROOT}/manual-test-runs/bin/yb-voyager`, and the skill also writes reports under `./manual-test-runs/...`. Neither `manual-test-runs/` nor this binary path is covered by any `.gitignore` (`git check-ignore manual-test-runs/bin/yb-voyager` → not ignored; no matching entry in `.gitignore` or `yb-voyager/.gitignore`).

Consequences:
- The compiled `yb-voyager` binary (~100 MB) shows up as untracked in `git status` and can be swept into a commit via `git add -A`.
- The skill's own Phase 1 reasons about the tree via `git diff`; leaving untracked artifacts in the working tree is noise it created.
- `SKILL.md:73` claims the report dir is "kept out of the repo tree unless the user says otherwise", but `./manual-test-runs/` is *inside* the repo tree — the code contradicts the doc.

Suggested fix: either add `manual-test-runs/` to `.gitignore` as part of this PR, or default `OUT`/the report dir to a location outside the repo (e.g. under `$TMPDIR`/`/tmp` or an explicit scratch dir) so ad-hoc runs never dirty the working tree. If keeping the in-tree default, update the `SKILL.md:73` wording to match.

---
_Generated by [Claude Code](https://claude.ai/code)_

File: ai/skills/manual-test-voyager/scripts/teardown.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+#
+#   PG_CONTAINER=mtv-pg-src  YB_CONTAINER=mtv-yb-tgt   KEEP_DATA=0
+set -euo pipefail
+PG_CONTAINER="${PG_CONTAINER:-mtv-pg-src}"
+YB_CONTAINER="${YB_CONTAINER:-mtv-yb-tgt}"
+
+for c in "$PG_CONTAINER" "$YB_CONTAINER"; do
+  if docker inspect "$c" >/dev/null 2>&1; then
+    echo "[teardown] removing $c" >&2
+    docker rm -f "$c" >/dev/null
```
Comment (makalaaneesh) [inline ai/skills/manual-test-voyager/scripts/teardown.sh:13]: [AI] **`KEEP_DATA` is documented as an override but never honored — containers are always force-removed.**

The header (line 5) advertises `KEEP_DATA=0` as an overridable env var, but the script never reads `KEEP_DATA`; the loop unconditionally runs `docker rm -f "$c"`. A user who sets `KEEP_DATA=1` to preserve the environment for inspection will still have both `mtv-*` containers destroyed — silently losing the exact state they wanted to debug. This directly undercuts `SKILL.md`'s Phase 4 note that teardown happens "unless the user wants the environment kept for inspection", which `KEEP_DATA` appears to be the mechanism for.

Suggested fix: honor the flag before removing, e.g.

```bash
KEEP_DATA="${KEEP_DATA:-0}"
if [[ "$KEEP_DATA" != "0" ]]; then
  echo "[teardown] KEEP_DATA set, leaving containers running" >&2
  exit 0
fi
```

Or, if keeping the data is not actually a supported feature, drop `KEEP_DATA=0` from the header comment so it doesn't promise behavior that isn't there.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Review by makalaaneesh — COMMENTED** [review]
[AI] Automated review — 2 findings (0 critical, 2 warning). See inline comments. Scope: new AI QA skill docs + helper shell scripts under `ai/skills/` (no Go product code). Scripts are otherwise clean (`set -euo pipefail` throughout).

---
_Generated by [Claude Code](https://claude.ai/code)_
